controllers.py

```diff
@@ -44,7 +43,31 @@ def busca_diretor():
          return jsonify(diretores_from_bd(diretor_escolhido))
      elif diretor_escolhido == None:
          return jsonify({"Erro": "Diretor não encontrado..."})
+# --------------------------------------------------------------------------
+@app.route("/generos", methods=['POST'])
+def inserir_genero():
+    genero = genero_from_web(**request.json)
+    if valida_genero(**genero):
+        insert_genero(**genero)
+        genero_inserido = get_genero(genero(['nome']))
```

Comment:
Agora que o insert retorna o id do item cadastrado, você pode fazer assim
```suggestion
        id = insert_genero(**genero)
        genero_inserido = get_genero(id)
```